core/src/main/java/com/graphhopper/routing/ev/Toll.java

```diff
@@ -35,11 +36,11 @@
 
     public static Toll find(String name) {
         if (name == null)
-            return NO;
+            return MISSING;
         try {
             return Toll.valueOf(Helper.toUpperCase(name));
         } catch (IllegalArgumentException ex) {
-            return NO;
+            return MISSING;
```

Comment:
Good idea, thanks!

Reply:
Do we even need this method?

Reply:
It does not seem so. Not sure if it is required for consistency reasons?